source/docs/essentials/create-a-ui-component.md

```diff
@@ -0,0 +1,307 @@
+---
+id: create-a-ui-component
+title: Create a UI Component
+---
+
+In Front Commerce we split our components in two categories: the **UI**
+components available in the `theme/components` folder, and the business
+components available in the `theme/modules` and `theme/pages` folders.
+
+<blockquote class="info">
+If you feel the need to understand why we went for this organization, feel
+free to refer to [React components structure](#TODO)
+first.
+</blockquote>
+
+In this documentation, you will learn how to build a UI Component. We will do so by
+creating our own. Front-Commerce’s core UI components follow the same principles
+and you could dive into the `node_modules/front-commerce/src/web/theme/components`
+folder to find examples into our source code.
+
```

Comment:
We could add a sentence here explaining that we chose to also keep Storybook mentioned in the process because it is for us part of the same workflow and link to the other page when it is done so people could have some background.